docker/config/excerpts/default-environment

```diff
@@ -18,14 +18,12 @@ PIPELINE_VM_MODE=True
 
 LOG_DIR=/pipeline/logs
 LOCAL_REPOSITORY_PATH=/home/openkim/
+LOCAL_DATABASE_PATH=/pipeline/db
 USE_FULL_ITEM_NAMES_IN_REPO=True
 
 # parent directory where things are run by default
 WORKER_RUNNING_PATH=/tmp
 
-# We install MD/MO/SM to the KIM API user collection
-KIM_API_USER_COLLECTION=/home/openkim/.kim-api/
```

Comment:
`KIM_API_USER_COLLECTION=/home/openkim/.kim-api/` was a special envvar that the KIM API used to inspect when figuring out where to install or load models but it may no longer use it (and it was also only one way to specify that information). As long as you've confirmed that models are installed where you need them to be and are accessible when doing things like `pipeline-run-pair`, this should be fine.

Reply:
@dskarls 

Confirmed that the current version of the API doesn't know anything about this variable:
```
ilia@ilia-xps:~/kim-api-2.3.0$ grep KIM_API_USER_COLLECTION README.md 
ilia@ilia-xps:~/kim-api-2.3.0$ find -type f | xargs grep KIM_API_USER_COLLECTION 
```